test/test.py

```diff
@@ -71,18 +71,32 @@ def test_score_combined(self):
         # Compare with a reference.
         self.assertTrue(_logs_equal(_LOG, _REF_UPDATED))
 
-if __name__ == '__main__':
-    print('Integration tests: ', end='')
-
-    # Log file not found test.
-    fs_py = ['python', os.path.join('src', 'facere_sensum', 'facere_sensum.py'),
-             '--log', 'llog.csv', 'update']
-    res = subprocess.run(fs_py, check=False, capture_output=True, text=True).stdout # nosec B603
-    if res == 'Log file \'llog.csv\' not found. Exiting.\n':
+def _test_integration(descr, args, ref):
+    '''
+    Run an integration test.
+    'descr' is test user description.
+    'args' command line arguments to use.
+    'ref' expected output.
+    '''
+    print(descr, end=': ')
+    res = subprocess.run(['python',
+                         os.path.join('src', 'facere_sensum', 'facere_sensum.py')] + args,
+                         check=False, capture_output=True, text=True).stdout # nosec B603
+    if res == ref:
         print('OK')
     else:
-        print('FAILED. Output:', res)
+        print('FAILED', 'Output:', '---', res, '---', 'Expected:', '---', ref, '---', sep='\n')
```

Comment:
Why multiple comma separated strings?

Also why not using [f-strings](https://docs.python.org/3.6/reference/lexical_analysis.html#f-strings)?

Reply:
The way Python 'print' works by default is printing arguments with spaces in between. Here I add sep='\n' to use new line instead of space, so this all is just a convenient way to print each argument in its own line. Could use f-strings as well, just a matter of personal preference. Suggest to keep as is.